packages/clickhouse-server.postinstall

```diff
@@ -26,6 +26,14 @@ if [ ! -f "/etc/debian_version" ]; then
 fi
 
 if [ "$1" = configure ] || [ -n "$not_deb_os" ]; then
+    # 'ii' means install and installed
+    # ('ic' for Installed but configuration incomplete)
+    # If it is the first installation, we might encounter the 'ic' (Incomplete) status when executing the postinst script
+    installed=$(dpkg -l | grep '^ii\s+clickhouse-server')
```

Comment:
Please check if it remains correct after my edits.

Reply:
Using single quotes is also incorrect, so I switched to using `grep -E`